src/qt/bitcoincore.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2014-2021 The Bitcoin Core developers
```

Comment:
The initial code was contributed by @laanwj back in [2014](https://github.com/bitcoin/bitcoin/pull/3493).